kuma/javascript/src/document-provider.jsx

```diff
@@ -89,6 +92,21 @@ export default function DocumentProvider(
                         window.scrollTo(0, 0);
                         setDocumentData(documentData);
                         body.style.opacity = '1';
+
+                        // Tell Google Analytics about this navigation.
+                        // We use 'dimension19' to mean client-side navigate
+                        ga('set', 'dimension19', 'Yes');
+
+                        // If the document data includes enSlug, or if the
+                        // document is in english, then pass the slug to
+                        // google analytics as dimension 17.
+                        if (documentData.enSlug) {
+                            ga('set', 'dimension17', documentData.enSlug);
+                        } else if (documentData.locale === 'en-US') {
+                            ga('set', 'dimension17', documentData.slug);
+                        }
+
+                        ga('send', 'pageload');
```

Comment:
I know very little about how we use `ga` events, so this is just a double check. You send a `pageview` event in the user provider component and in a comment within that component, you mention sending a `pageview` event in the document provider component, but here you send a `pageload` event. I assume that's what you wanted, but just wanted to make sure.

Reply:
I'll double-check and make sure I'm using the right string in both the comments and the code. Thanks for catching that!